src/offices/_single.html

```diff
@@ -100,22 +101,30 @@ <h2 class="u-visually-hidden">
             {% if resource.icon %}
             <div class="media_image-container
                         media_image-container__wide-margin">
-                <img class="media_image u-centered-on-mobile"
-                     width="150"
+                <img class="media_image
```

Comment:
Indeed, none of the "office"/operational resource pages are using a subscription block.

Any "office" page that isn't linked through the mega menu or footer -- and is not on the proposed site map -- should be deleted. I didn't realize that OFE page was technically live -- it should be removed.